src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java

```diff
@@ -639,7 +639,7 @@ void should_return_status_code_502_when_camunda_service_is_down() throws Excepti
                 jsonPath("$.status").value(502),
                 jsonPath("$.detail").value(
                     "Downstream dependency did not respond as expected "
-                        + "and the request could not be completed.")
+                        + "and the request could not be completed. Message from downstream system: null")
```

Comment:
Is there no error from the downstream system here?  There are a few more examples below

Reply:
I thought the same Ravi. I found out during the debugging that "null" is the message coming from the downstream system. I think its because of the line 613, they are not setting any message there. Shall i create a proper exception and mock it Ravi?

Reply:
I am making the chnages.